src/main/java/net/neoforged/neoforge/client/event/ContainerScreenEvent.java

```diff
@@ -8,6 +8,7 @@
 import net.minecraft.client.gui.GuiGraphics;
 import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
 import net.neoforged.bus.api.Event;
+import net.neoforged.bus.api.ICancellableEvent;
```

Comment:
Pointless change, please revert this

Reply:
it's just a javadoc fix. I don't think we need to go head over heels for this.

Reply:
Random unrelated fixes really do not help with tracking the history of changes in a file.

Reply:
ok